.github/workflows/code.yml

```diff
@@ -170,11 +170,45 @@ jobs:
       - name: Test cli works in runtime image
         run: docker run ${{ env.IMAGE_REPOSITORY }} --version
 
+  conda:
+    needs: [lint, dist, test]
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/download-artifact@v3
+
+      - name: Install grayskull
+        run: pip install grayskull
+
+      - name: Create Conda recipe
+        run: |
+          mkdir -p conda/recipe
+          grayskull pypi -o conda/recipe dist/*.tar.gz
+
+      - name: Install conda-build
+        run: conda install -y conda-build
```

Comment:
I'm guessing this can't be run under pipx too?

Reply:
Though hypothetically it should, I can't seem to get it to work. It would seem some of the necessary dependencies are only published to PyPI in source format so python headers are required to build the module

Reply:
Never mind, even jumping past that hurdle I am met with:

> ERROR: The install method you used for conda--probably either `pip install conda`
or `easy_install conda`--is not compatible with using conda as an application.
If your intention is to install conda as a standalone application, currently
supported install methods include the Anaconda installer and the miniconda
installer.  You can download the miniconda installer from
https://conda.io/miniconda.html.

Reply:
Fair enough, I suppose we should use conda for conda-build, makes sense...